### ISSUE TITLE
Automate mesos/gce cluster deployment

### ISSUE THREAD

**Issue by karlkfi** [mesosphere/kubernetes-mesos#445]
Now that k8s-mesos integration is upstreamed, one of the next steps is to automate deployment of a mesos cluster on GCE.

This may not be high priority, once DCOS can be easily deployed on GCE.

**Comment by jdef** [comment]
agreed, low priority. we should probably have a dcos/gce cluster provider
first.
